pcsx2/R5900.cpp

```diff
@@ -56,7 +56,14 @@ static const uint eeWaitCycles = 3072;
 
 bool eeEventTestIsActive = false;
 
-u32 eeloadMain = 0;
+u32 g_eeloadMain = 0, g_eeloadExec = 0, g_osdsys_str = 0;
+const int kMaxArgs = 16; /* I don't know how much space for args there is in the memory block used for args in full boot mode,
```

Comment:
I prefer is the entire comment is above kMaxArgs.

Reply:
Sure, makes sense to me; I will change that.  (Going away for some hours, but I will reply to other comments when I'm back.)